packages/server/src/api/index.ts

```diff
@@ -2700,6 +2700,34 @@ router.get('/user/read-later/unread/:size/:cursor', async (req, res) => {
  * Get user activities of the logged in user with pagination ( latest first )
  */
 router.get('/user/article-activities/:size/:cursor', async (req, res) => {
+    const { size, cursor } = req.params;
+
+    if (+cursor <= 0 || +size <= 1) return res.status(400).json({ error: 'cursor cannot be less than 0 and/or size cannot be less than 1' });
+
+    const prisma = PrismaClientSingleton.prisma;
+
+    try {
+        const userActivities = await prisma.userActivity.findMany({
+            skip: 1,
```

Comment:
We cannot use the skip. Sorry about this

Reply:
[Here's](https://www.prisma.io/docs/concepts/components/prisma-client/pagination#do-i-always-have-to-skip-1) something regarding this

Reply:
That is a bug of skip ( we are actually querying the db with sorting and taking the ( size ) elements at a time.

Reply:
So do you want to include cursor?

Reply:
Yes!